plottable.css

```diff
@@ -120,8 +120,45 @@ svg.plottable {
   fill: aliceblue;
   opacity: .7;
   stroke: #C0C8FF;
+}
+
+/* SelectionBox */
+.plottable .selection-box-layer .selection-area {
+  fill: black;
+  fill-opacity: 0.03;
+  stroke: #CCC;
+}
```

Comment:
Want to sync a designer to see what the selection box should look like?  I'm unsure about a gray? box being the selection area